report/firefox_code_coverage/codecoverage.py

```diff
@@ -11,26 +11,34 @@
 import tempfile
 import time
 import warnings
+from datetime import timedelta
+from pathlib import Path
 
+import magic
 import requests
 
 from firefox_code_coverage import taskcluster
 
 FINISHED_STATUSES = ["completed", "failed", "exception"]
 ALL_STATUSES = FINISHED_STATUSES + ["unscheduled", "pending", "running"]
 STATUS_VALUE = {"exception": 1, "failed": 2, "completed": 3}
+TASKCLUSTER_SRC_DIR = "/builds/worker/workspace/build/src/"
 
 GRCOV_INDEX = "gecko.cache.level-3.toolchains.v3.linux64-grcov.latest"
 GRCOV_ARTIFACT = "public/build/grcov.tar.xz"
 
 
 def is_taskcluster_loaner():
-    return "TASKCLUSTER_INTERACTIVE" in os.environ
+    return os.environ.get("TASKCLUSTER_INTERACTIVE") == "true"
 
 
 def get_task(branch, revision):
     index = taskcluster.get_service("index")
-    task = index.findTask(f"gecko.v2.{branch}.revision.{revision}.firefox.decision")
+    task = index.findTask(
+        "gecko.v2.{branch}.revision.{revision}.firefox.decision".format(
```

Comment:
nit: f-string here and few others in this patch

Reply:
Because we are stuck on python 3.5 as that's the version in use in Taskcluster build tasks